src/containers/CreatePipelineResource/CreatePipelineResource.js

```diff
@@ -221,11 +213,7 @@ export /* istanbul ignore next */ class PipelineResourcesModal extends Component
     const stateVar = 'type';
     const stateValue = e.selectedItem.text;
     this.setState(prevState => {
-      if (stateValue === 'Git') {
-        gitSource = true;
-      } else {
-        gitSource = false;
-      }
+      gitSource = stateValue === 'Git';
```

Comment:
remove gitsource on line 212 and just define it here ?

Reply:
done